cockpit/src/main/java/com/polidea/cockpit/paramsedition/viewholder/StringParamViewHolder.kt

```diff
@@ -0,0 +1,30 @@
+package com.polidea.cockpit.paramsedition.viewholder
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import android.widget.EditText
+import com.polidea.cockpit.R
+import com.polidea.cockpit.core.CockpitParam
+
+class StringParamViewHolder(view: View) : ParamBaseViewHolder<String>(view) {
+
+    private val value: EditText = view.findViewById(R.id.cockpit_string_param_value)
+
+    init {
+        value.addTextChangedListener(object : TextWatcher {
+            override fun afterTextChanged(s: Editable?) {}
```

Comment:
Maybe we can implement our own CockpitTextWatcher, so inside init block we'd only need to override onTextChanged?

Reply:
Same as above